MSBuild.Caching/MSBuild.Caching.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <TargetFrameworks>net472;net6.0</TargetFrameworks>
+    <LangVersion>12.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.0.1" PrivateAssets="All" ExcludeAssets="runtime" />

```

Comment:
@bording I cherry picked this from https://github.com/adamralph/minver/pull/989

What determines the version of this package that should be referenced?

Reply:
My understanding is that the package version corresponds to the MSBuild version, so you would want to reference the version of the package that corresponds to the version of MSBuild that shipped in the oldest .NET SDK you're trying to support.

Since this is targeting `net6.0`, that would correspond to the version of MSBuild that ships in the 6.0.100 SDK.

Reply:
Thanks. On that assumption, as far as I can tell (it isn't easy), 17.0.1 is the right version.

Reply:
In the future you can reference this chart we made to help answer this question: https://aka.ms/dotnet/matrixofpaine

Reply:
@baronfel that's very useful, thanks!

Reply:
@baronfel so useful in fact, that I decided to bake it into the source — https://github.com/adamralph/minver/pull/1047/files